src/meta_utils.py

```diff
@@ -0,0 +1,528 @@
+            print("NaN values found, dropping rows containing NaNs...")
+            for df in [df1, df2]:
+                df.dropna(axis = 0, subset = [label], inplace = True)
+
+            print(f"Adjusted dataframe shapes : {df1.shape} , {df2.shape}")
+            # Take the intersection of indices b/twn two dataframes after dropping NaNs and subset
+            print(f"Taking index intersection of adjusted indices...")
+            indices = df1.index.intersection(df2.index)
+            df1 = df1.loc[indices, :]
+            df2 = df2.loc[indices, :]
+
+        # (3) Check that ids are corresponding
+        if (df1.plotid != df2.plotid).all():
+            raise AssertionError("IDs are not corresponding.")
+        
+        print("Loading and checking dataframes complete!")
+        return df1, df2
+
+def load_dataframes(
+        path_fn : Callable[[str], str], 
+        completed_date : Optional[str] = None,
+        final_date : Optional[str] = None
+    ) -> Tuple[pd.DataFrame, ...]:
+    """ Loads labeling CSVs to dataframe.
+    
+    Args:
+    
+    Returns:
+
+    """
+
+    if (completed_date is not None) and (final_date is not None):
+        print("{:^61}\n{}".format("Loading dataframes from file...", "-" * 59))
+        # Dataframes @ completed date for set 1 and 2
+        cdf1 = pd.read_csv(path_fn("set-1", completed_date))
+        cdf2 = pd.read_csv(path_fn("set-2", completed_date))
+        # Dataframe @ final date 
+        #   -> Arbitrarily choose "set-1", both sets are in agreement by this point. 
+        fdf = pd.read_csv(path_fn("set-1", final_date))
+
+        return check_dataframes(cdf1, cdf2, fdf)
+
+    else:
+        print("{:^53}\n{}".format("Loading dataframes from file...", "-" * 51))
+        # Dataframes @ completed date for set 1 and 2
+        cdf1 = pd.read_csv(path_fn("set-1"))
+        cdf2 = pd.read_csv(path_fn("set-2"))
+
+        return check_dataframes(cdf1, cdf2)
+
+def compute_area_change(year_1_label : str, year_2_label : str) -> str:
+    """ Computes planting change. """
+
+    match = {
+        ("Planted", "Planted") : "Stable P",
+        ("Not planted", "Not planted") : "Stable NP",
+        ("Planted", "Not planted") : "P loss",
+        ("Not planted", "Planted") : "P gain",
+    }
+
+    return match[year_1_label, year_2_label]
+
+
+def compute_disagreements(df1 : pd.DataFrame, df2 : pd.DataFrame, area_change : bool = False) -> pd.Series:
+    """ Computes disagreements between labeler sets. """
+    
+    if area_change:
+        print("\n{:^61}\n{}".format("Computing disagreements...", "-"*59))
+        disagreements = (df1["area_change"] != df2["area_change"])
+    else:
+        print("\n{:^53}\n{}".format("Computing disagreements...", "-"*51))
+        disagreements = (df1["crop_noncrop"] != df2["crop_noncrop"])
+    
+    print(f"Disagreements between labeler sets 1 and 2 : {disagreements.sum()}")
+    return disagreements
```

Comment:
nit: Just using the column name as the parameter is less restrictive and can condense this a bit
```python
def compute_disagreements(df1 : pd.DataFrame, df2 : pd.DataFrame, column_name: str) -> pd.Series:
    """ Computes disagreements between labeler sets. """
    disagreements = (df1[column_name] != df2[column_name])
    print(f"Disagreements between labeler sets 1 and 2 : {disagreements.sum()}")
    return disagreements
```
Writing this out I feel it might be cleaner to write 
```python
disagreements = (df1[column_name] != df2[column_name])
```
where it is used directly.